modules/create-vm/pkg/utils/parse/clioptions.go

```diff
@@ -35,6 +36,7 @@ type CLIOptions struct {
 	RunStrategy               string            `arg:"--run-strategy,env:RUN_STRATEGY" help:"Set run strategy to vm"`
 	Output                    output.OutputType `arg:"-o" placeholder:"FORMAT" help:"Output format. One of: yaml|json"`
 	Debug                     bool              `arg:"--debug" help:"Sets DEBUG log level"`
+	Virtctl                   string            `arg:"--virtctl,env:VIRTCTL" placeholder:"VIRTCTL" help:"Specifies a virtctl create vm params to be used to create VirtualMachine."`
```

Comment:
`s/params/command/`?

Reply:
In this task param should be specified only the parameters, that will be used with the virtctl command like this for example `--name vm-test --run-strategy Always`. So maybe it is beter to use params instead of a command?

Reply:
If you could improve the help text that's fine.